### ISSUE TITLE
Limitations of income data for measuring affluence

### ISSUE THREAD

**Issue by w4rner** [uchicago-computation-workshop/luis_bettencourt#17]
In 1.3 of the Supplementary Online Material to the 'Statistics of Neighborhoods' paper in which you discuss limitations of the income data, you wisely point out that the household income data lacks information on the right-tail of the income distribution (which is thought to be 'fat') and conclude that we may therefore underestimate the variance in rich neighborhoods, such as Manhattan's Upper East Side.

Would wealth be a more relevant operationalization of affluence than income when considering neighborhood selection, especially in rich neighborhoods where property is often a wealth investment? If so, given that wealth inequality is likely more extreme than income inequality, do you think we may be underestimating economic variance within rich neighbourhoods even more than you suggest?

**Comment by lmbett** [comment]
Yes: great question. I agree with where you are going with the question but as you know, wealth is even harder to measure. I think one could at least look at real estate values for which there is some data, I'd like to do this, but have not yet. It may also be doable more generally in other nations.

**Comment by w4rner** [comment]
Thank you very much for your response! Looking at real estate values seems like a fantastic idea.

Related question: on the other end of the distribution, in poor neighbourhoods do you think that economic segregation may actually be overestimated in your analysis, given that income survey data may not account for the equalizing effects of economic benefits such as social security? (This may partly be an empirical question as to whether these benefits are actually taken up by the extremely poor).
